R/source.R

```diff
@@ -22,11 +22,26 @@ source_package <- function(path) {
   load_pkg_dependencies(path)
 
   paths <- package_files(path)
-  lapply(paths, sys.source, envir = env, keep.source = FALSE)
+  lapply(paths, sys_source, envir = env)
 
   env
 }
 
+sys_source <- function(file, envir = baseenv()) {
```

Comment:
Unfortunately, it seems like not possible to use `sys.source()`, because it doesn't allow `file()` connection as the input. Here's the first line source code of `sys.source()`:

```
    if (!(is.character(file) && file.exists(file))) 
        stop(gettextf("'%s' is not an existing file", file))
```